src/nix-daemon/nix-daemon.cc

```diff
@@ -557,7 +557,8 @@ static void performOp(TunnelLogger * logger, ref<Store> store,
                     ;
                 else if (trusted
                     || name == settings.buildTimeout.name
-                    || name == "connect-timeout")
+                    || name == "connect-timeout"
+                    || (name == "builders" && value == ""))
```

Comment:
@dtzWill would you mind sending a PR adding docs for this?

Reply:
Don't mind at all, but not sure how to edit the docs.  Will take a look at that later today, probably should bite the bullet! If you find yourself wondering when I'm going to do it please bug me about it ;).

But sure thing, and thanks for all your documentation work! (including ensuring documentation exists for new things, like here!)